Description: add linking to SSL
Author: Dariusz Dwornikowski <dariusz.dwornikowski@cs.put.poznan.pl>
Forwarded: no
--- a/configure.ac
+++ b/configure.ac
@@ -9,7 +9,6 @@
 
 ### Checks for programs.
 AC_PROG_CC
-
 ### Get canonical host
 AC_CANONICAL_HOST
 PLATFORM="unknown"
@@ -214,7 +213,7 @@
 AC_CHECK_HEADERS([ncurses.h], [], [])
 
 ### Default parameters
-AM_CFLAGS="-Wall -Wno-deprecated-declarations"
+AM_CFLAGS="-Wall -Wno-deprecated-declarations -lssl -lcrypto"
 AS_IF([test "x$PACKAGE_STATUS" = xdevelopment],
     [AM_CFLAGS="$AM_CFLAGS -Wunused -Werror"])
 AM_CPPFLAGS="$AM_CPPFLAGS $glib_CFLAGS $curl_CFLAGS $libnotify_CFLAGS"
