Description: drop unneeded dependency on libssl
Author: Dariusz Dwornikowski <dariusz.dwornikowski@cs.put.poznan.pl>
Forwarded: no
--- a/configure.ac
+++ b/configure.ac
@@ -74,10 +74,6 @@
 AC_CHECK_LIB([resolv], [res_query], [],
     [AC_CHECK_LIB([resolv], [__res_query], [],
         [AC_MSG_ERROR([libresolv is required for profanity])])])
-PKG_CHECK_MODULES([openssl], [openssl], [],
-    [AC_MSG_ERROR([openssl is required for profanity])])
-AM_CPPFLAGS="$AM_CPPFLAGS $openssl_CFLAGS"
-LIBS="$LIBS $openssl_LIBS"
 
 # TODO: autodetect of XML parser libstrophe linked with
 CFLAGS_RESTORE="$CFLAGS"
