Subject: fix spelling in notifier.c
Author: Dariusz Dwornikowski <dariusz.dwornikowski@cs.put.poznan.pl>
Forwarded: https://github.com/boothj5/profanity/pull/413
--- a/src/ui/notifier.c
+++ b/src/ui/notifier.c
@@ -121,7 +121,7 @@
 {
     GString *message = g_string_new("Subscription request: \n");
     g_string_append(message, from);
-    _notify(message->str, 10000, "Incomming message");
+    _notify(message->str, 10000, "Incoming message");
     g_string_free(message, TRUE);
 }
 
